src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -117,7 +118,7 @@ public virtual void InstallPlatformSpecificSkeletonDependencies(StringBuilder st
                 $"--output {tarFile} >/dev/null 2>&1")
                 .AppendLine("else")
                 .AppendLine(
-                $"curl -D headers.txt -SL \"{sdkStorageBaseUrl}/{platformName}/{platformName}-$DEBIAN_FLAVOR-{version}.tar.gz\" " +
+                $"curl -D headers.txt -SL \"{sdkStorageBaseUrl}/{platformName}/{platformName}-$DEBIAN_FLAVOR-{version}.tar.gz{sdkStorageAccountAccessToken}\" " +
```

Comment:
It looks to me like we are getting the string value of the SAS token here, and actually printing it out to the build script here. Probably more secure to make sure that we don't actually print it into the file. 

In cases where partners use our image in a multi-stage build, they usually copy the oryx files over to their final image which would include the sas token. I think that it would be better to figure out a way to make sure this script just uses the `ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN` env var during execution without printing the string here.

Reply:
For example, just using `$ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN` here instead at least works to keep it from being printed in my limited testing of $DEBIAN_FLAVOR